lib/github_changelog_generator/octo_fetcher.rb

```diff
@@ -456,10 +455,21 @@ def iterate_pages(client, method, *arguments, parent: nil, **options)
     #
     # @return [Object] returns exactly the same, what you put in the block, but wrap it with begin-rescue block
     # @param [Proc] block
-    def check_github_response(&block)
-      Retriable.retriable(retry_options, &block)
+    def check_github_response
+      yield
     rescue MovedPermanentlyError => e
       fail_with_message(e, "The repository has moved, update your configuration")
+    rescue Octokit::TooManyRequests => e
+      resets_in = client.rate_limit.resets_in
```

Comment:
🔍 I learned! resets_in (a time) is even provided by Octokit, thanks for finding this out!

Reply:
You are most welcome!